views/default/tab_client_actions.pdt

```diff
@@ -14,7 +14,7 @@
     ?>
 
     <div id="vultr_status_badge">
-        <span class="label label-<?php $this->Html->_($status_class); ?>">
+        <span class="badge badge-<?php $this->Html->_($status_class); ?>">
```

Comment:
This badge needs some margin on the bottom, it is right up against the header below it.